src/gatsby-theme-carbon/components/Footer.js

```diff
@@ -4,33 +4,29 @@ import Footer from 'gatsby-theme-carbon/src/components/Footer';
 const Content = () => (
   <>
     <p>
-      By importing the <strong>Footer</strong> component from
-      gatsby-theme-carbon, we can supply our own props.
+      Have questions? Open an issue in <a href="https://github.com/carbon-design-system/carbon-for-ibm-website/issues/new/choose">Github</a>.
     </p>
     <p>
-      The default export from a shadowed component will replace that component
-      in the theme.
-    </p>
-    <p>
-      <a href="https://www.gatsbyjs.org/docs/themes/api-reference/#component-shadowing">
-        More about component shadowing
-      </a>
+      Last updated November 12th, 2019
```

Comment:
@JennySanchez is the plan to manually update this date? Not sure if there is a way to grab latest changed date and time, we might have to build a small custom component for that.